src/main/java/UseCase/Login.java

```diff
@@ -1,21 +1,38 @@
 package UseCase;
 
+import Database.LoginUserAccessInterface;
 import InputBoundary.loginInputBoundary;
 import InputData.loginInputData;
 import OutputBoundary.loginOutputBoundary;
 import Database.GetUserAccessInterface;
+import OutputData.loginOutputData;
 
 public class Login implements loginInputBoundary{
     private final loginOutputBoundary loginPresenter;
-    private final GetUserAccessInterface allUsers;
+    private final LoginUserAccessInterface loginUsers;
+    private final GetUserAccessInterface getUsers;
 
     public Login(loginOutputBoundary loginPresenter) {
-        this.allUsers = Database.userList.getInstance();
+        this.loginUsers = Database.userList.getInstance();
+        this.getUsers = Database.userList.getInstance();
         this.loginPresenter = loginPresenter;
     }
 
     @Override
-    public void login(loginInputData loginID) {
-        //TODO: currently implementing
+    public boolean login(loginInputData loginID) {
+        if (!loginID.isArtist()){
```

Comment:
I believe we don't need to check whether loginID is an artist because login use case is the same for both type of users, the difference only occurs in the view, which is why we will take care of that in the view layer instead of here to avoid duplicate code